.gflows/workflows/build-publish/build-publish.template.yml

```diff
@@ -32,6 +33,10 @@
   #@  jobs["scan-code"] = generate_scan_code_net_job(data.values.scan_code_net, data.values)
   #@ end
 
+  #@ if hasattr(data.values,"scan_code"):
```

Comment:
Would be nice to update sample as well with this new `scan_code`

Reply:
Added